### DISCUSSION TITLE
Valor default no PoPageDynamicEditField

### DISCUSSION THREAD

**Discussion by leandrodefaria** [po-ui/po-angular#1145] [Tire suas dúvidas] [answered]
Pessoal, boa noite.

Começando a aprofundar no po ui, então tenho uma duvida. Existe a possibilidade de incluir um valor default em um field do PoPageDynamicEditField. Ex:

> { property: 'type', label: "Tipo", gridColumns: 6, options: [
        { label: 'Pessoa Fisica', value: 1,},
        { label: 'Pessoa Juridica', value: 2 }
      ],

Queria ao abrir a pagina a opção "Pessoa Fisica" já estivesse marcado.

Obrigado

**Comment by leandrodefaria** [comment, marked as answer]
Galera,

Resolvi com a property valide e atualizei o novo valor

> { property: 'name', label: 'NOME', validate: this.onChangeField.bind(this), required: true, gridColumns: 6, gridSmColumns: 12 }

> 
    private onChangeField(changeValue: any): PoDynamicFormFieldValidation {

    let retChange = {};
 
      retChange = { value: 'TESTE', field: { property: 'name'}, focus: true};   
     return retChange;
  }

**Reply by wsteixeira** [reply]
Olá,

Vale salientar que o evento "validate" só é disparado quando o valor do campo é alterado!!
